docs/ClinicaConferences.md

```diff
@@ -1,5 +1,12 @@
 # Clinica at Conferences
 
+## Neuro OpenScience Workshop 2023
+
+Clinica was presented at the [Neuro OpenScience Workshop](https://open-neuro.org) held at the [ICM](https://icm-institute.org/) on November 22-23 2023.
+You can find on [this page](FIXME) the slides of the presentation.
```

Comment:
@nburgos do you have a public link for the slides you used for the presentation ?

Reply:
Yes! https://ninonburgos.com/wp-content/uploads/2023/12/2023.11.22-NOW-Clinica.pdf

Reply:
Thanks ! 🙏